src/Api/ApiServiceProvider.php

```diff
@@ -101,6 +101,7 @@ protected function registerNotificationSerializers()
             'discussionRenamed' => BasicDiscussionSerializer::class
         ];
 
+        // Deprecated in beta 15, remove in beta 16
```

Comment:
Another place where the singleton approach might be better: best to avoid having this logic in multiple places if we can aboid it.